remoulade/utils.py

```diff
@@ -14,14 +14,15 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from .errors import NoScheduler
 
 global_scheduler = None
 
 
 def get_scheduler():
     global global_scheduler
     if global_scheduler is None:
-        raise ValueError("Scheduler not found, are you sure you called set_scheduler(scheduler) ?")
+        raise NoScheduler("Scheduler not found, are you sure you called set_scheduler(scheduler) ?")
```

Comment:
why did you remove the test ? you should add a test, when there is no scheduler and verify the result.